README.md

```diff
@@ -0,0 +1,10 @@
+# UML CCDC 2025
+Contributions should be organized as follows:
+
+1. Incident and Inject responses should be stored in their associated folder in [0-Documentation/Injects-Incidents/](./0-Documentation/Injects-Incidents/).
+2. Printed materials should be stored in [0-Documentation/](./0-Documentation/Printables/).
+3. Platform and service related materials should be stored in their respective folder. For example AWS should have it's related materials stored in [Platform-AWS/](./Platform-AWS/).
+   * A copy of all scripts should be stored in a folder named after the topic they are associated with in the `Platform/0-Scripts` for example AWS CLI scripts would have a copy stored in [Platform-AWS/0-CLI/](./Platform-AWS/0-CLI/) directory. This is done for quick and easy access.
```

Comment:
Could you elaborate on the "copy" directory. Its not the clearest, in my opinion. 
- I am assuming topic are components of a larger technology, Ex. (AWS/MFA) MFA is the topic.

Are you suggesting we have a larger more general directory where all the scripts are located (for easy access)? While in the original script directory, it has more details relating to how to use/operate the script and during what conditions to use it?

Reply:
Yes that is what it would be.